src/app/login/login.component.html

```diff
@@ -24,18 +24,17 @@ <h4 class="version">v{{version}}</h4>
         <br/>
```

Comment:
why did you changed to login file ? 
its ok and NOT MERGE BLOCKING !  but i want to know why :D

Reply:
It gave me errors when i pulled master br, so I commented this part of code just to continue my work in the form and refer to this error after I finish to fix it. (Stupid solution  :D ).